Added documentation of publish flag feature (#34)
GovTechSG/techpass-tenant-docs: docs/applications.md

> Unchecking this field will disable ALL user sign in to the tenant service.

Reviewer: If you clear the selection, it will disable...

### User Assignment Required

> This property determines if a user needs to be granted access before he/she is allowed to sign into your app. Leaving this field uncheck means that **ALL** TechPass users will be able to sign into your app.

Reviewer: Generally, we avoid genders and Check to be documented as "select".

 so suggestion:

This indicates if users need access for signing into the app. If you do not select this option,  all users with an active TechPass account will be able to sign in to the app.

Reviewer 2: great suggestion.. i did have to re-read 1st part though. I've rephrased a little:
Select this option to grant application access for a subset of users. If you do not select this option, all users with an active TechPass account will be able to sign in to the app.

Author: Should I write this?:
> Select **User Assignment Required** to grant application access for a subset of users. If you do not select this option, all users with an active TechPass account will be able to sign in to the app.

Because @SenthilKumarSivakami suggested "field name in bold" in other comments as well so I think it should be better if we keep it consistent

> Careful consideration is required if you wish the alter the value of this field. Find out more about to how design your [Access Model](resources#concepts) with TechPass.

Reviewer: Can we specify what factors are to be considered here? what is the implication of changing the value?

Reviewer 2: your suggestion above should suffice

> If you left this field checked, you will need to grant access to users that you want to allow into your app. See [Assigning Access to your App](#assigning-access-to-your-app) for more info.

Reviewer: If you don't select this option, you need to grant access to the required users allowing them to sign in to the app. 

is there a name for this field or option? it is better to specify that in bold.

Reviewer 2: see line 49. it's a continuation from it

### Mark As Published

> Checking this field will publish your application on TechPass portal.

Reviewer: Select <field name in bold> to publish your application on the TechPass Portal. 

i see we are using apps and applications interchangeably, right?

Reviewer 2: ```suggestion
Checking **Mark As Published** will publish your application on TechPass portal.
```
something like that?

yes, noticed the apps and applications terms.. I suggest to spell it out

> Leave as unchecked if your app should not be displayed to users e.g. apps for testing etc.

Reviewer: can we lose this or if you want to give this as a tip , can this be 

If your application is in the testing phase, do not select "filed name" is not selected.

and can be appended to the previous line.

Reviewer 2: hmmm... we should include this tip.

maybe can add a note to avoid publishing testing applications
